build/musicbrainz-dev/scripts/fetch-dump.sh

```diff
@@ -98,6 +98,59 @@ then
 	exit 64 # EX_USAGE
 fi
 
+# Show information about signing up for data use
+
+if [[ $TARGET != sample &&
+	! -a "$DB_DUMP_DIR/.for-commercial-use" &&
+	! -a  "$DB_DUMP_DIR/.for-non-commercial-use"
+]]
+then
+	prompt=$(cat <<-EOQ
+		The data you are about to download is provided by the MetaBrainz Foundation.
+		Are you planning to use this data for commercial or business purposes?
+		(y/n)
+	EOQ
+	)
+	read -e -p "$prompt " -r
+	while [[ ! ${REPLY:0:1} =~ [YNyn] ]]
+	do
+		read -e -p "Invalid reply. Yes or no? " -r
```

Comment:
If we're going to ask `Yes or no?` here with full words then we should also allow answering with the full word. Otherwise, put y/n here too I guess?

Reply:
We do allow `y` `Y` `Yes` `yes` `YES` and even `Yay!`

Reply:
Oh, wait, so it just checks the first letter is y vs n? So "Yikes!" is yes, but "Naturally!" is no? :D (that actually seems good enough to me, I guess I didn't pay enough attention).